quickstart/binaries.markdown

```diff
@@ -3,29 +3,24 @@ title: Binaries - CVMFS and RPMs
 layout: main
 ---
 
-Automatic builds
-================
+We provide builds of the O² ecosystem on CVMFS and via RPMs. 
 
-We provide builds of the O² ecosystem on CVMFS and via installable RPMs. Every night (Geneva time)
-a new O² build is produced and published on `alice-nightlies.cern.ch`. RPMs are produced on-demand
-instead.
+# CVMFS
 
+## On lxplus
+You can run our O² CVMFS builds from `lxplus7.cern.ch` to be on the safe side: CVMFS is immediately
+available and you can start running in seconds. 
 
-Run from CVMFS
-==============
+## On your own machine
 
-You can run our O² CVMFS builds from `lxplus7.cern.ch` to be on the safe side: CVMFS is immediately
-available and you can start running literally in seconds. If you are using your own machine, bear in
-mind that the following operating systems should be supported:
+The following operating systems should be supported:
 
   * RHEL 7 (and newer)
   * Ubuntu 16.04/Debian 7 (and newer)
 
-macOS is not supported.
-
+MacOS is not supported.
```

Comment:
moreover the following OS:

* Ubuntu 18.04 / Debian 7 (and newer)
* macOS Catalina

are supported on a best effort basis for development and analysis.